src/changes-backwards-compatible.apib

```diff
@@ -6,6 +6,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added `lost_reason` to the response of `deals.info`.
```

Comment:
PP: have #494 as the base branch and add `deals.info` to the addition added there.